pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/plugin/PythonPlugin.java

```diff
@@ -36,6 +36,7 @@
 
 import static com.linkedin.gradle.python.util.StandardTextValues.CONFIGURATION_BOOTSTRAP_REQS;
 import static com.linkedin.gradle.python.util.StandardTextValues.CONFIGURATION_BUILD_REQS;
+import static com.linkedin.gradle.python.util.StandardTextValues.CONFIGURATION_FLAKE8;
```

Comment:
I chose flake8 because we currently support only flake8. For other checkers, we should probably use different configurations, so they can get their own deps.